modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -599,10 +709,12 @@ PERF_TEST_P_(Cart2PolarPerfTest, TestPerformance)
 
 PERF_TEST_P_(CmpPerfTest, TestPerformance)
 {
-    CmpTypes opType = get<0>(GetParam());
-    cv::Size sz = get<1>(GetParam());
-    MatType type = get<2>(GetParam());
-    cv::GCompileArgs compile_args = get<3>(GetParam());
+    compare_f cmpF;
+    CmpTypes opType;
+    cv::Size sz;
+    MatType type;
```

Comment:
Please add initialization with default values. See example in the `MulPerfTest` case (above).

Reply:
Applied, thanks